test/minErrSpec.js

```diff
@@ -143,4 +162,36 @@ describe('minErr', function() {
     expect(testError('acode', 'aproblem', 'a', 'b', 'value with space').message)
       .toMatch(/^[\s\S]*\?p0=a&p1=b&p2=value%20with%20space$/);
   });
+
+  it('should slice error reference URL in the message if it exceeds url max length', function() {
+    var a = createDummyString(2000);
+
+    var myError = testError('26', 'a when default=2000 is {0}', a);
+    var url = extractUrlFromErrorMessage(myError.message);
+    expect(url.length).toBe(2000);
+    expect(errorHandlingConfig().urlMaxLength).toBe(2000);
```

Comment:
These assertions are important to test that the default value is 2000 and doesn't change.